packages/babel-core/src/util.js

```diff
@@ -116,7 +116,9 @@ export function shouldIgnore(
   ignore: Array<RegExp | Function> = [],
   only?: Array<RegExp | Function>,
 ): boolean {
-  filename = slash(filename);
+  if (path.sep !== "/") {
```

Comment:
We probably can safely remove this check, as `slash()` is also always doing the replacement, not matter what `path.sep` is set to.
https://github.com/sindresorhus/slash/blob/master/index.js